taxonomies/labels.txt

```diff
@@ -22662,18 +22662,18 @@ web:en: https://www.canada.ca/en/health-canada/services/food-nutrition/nutrition
 web:fr: https://www.canada.ca/fr/sante-canada/services/aliments-nutrition/etiquetage-nutritionnel/devant-emballage.html
 
 < en:Canadian front-of-package nutrition symbol
-en: High in Sat fat Sugars and Sodium (Health Canada)
-fr: Élevé en Gras sat., Sucres and Sodium (Santé Canada)
-image:xx: high-in-saturated-fat-sugars-sodium-health-canada-eleve-en-gras-satures-sucres-sodium-sante-canada.171x90.svg
-image:en: high-in-saturated-fat-sugars-sodium-health-canada.120x90.svg
-image:fr: eleve-en-gras-satures-sucres-sodium-sante-canada.120x90.svg
+en: High in Sat fat Sugars and Sodium (Health Canada), High in Saturated fat Sugars and Sodium (Health Canada)
+fr: Élevé en Gras sat. Sucres et Sodium (Santé Canada), Élevé en Gras saturés Sucres et Sodium (Santé Canada)
+image:xx: high-in-sat-fat-sugars-sodium-health-canada-eleve-en-gras-sat-sucres-sodium-sante-canada.171x90.svg
```

Comment:
high-in-sat-fat-sugars-sodium-health-canada-eleve-en-gras-sat-sucres-sodium-sante-canada.171x90.svg does not exist.